src/main/java/oshi/software/os/linux/LinuxFileSystem.java

```diff
@@ -37,20 +39,61 @@
  * device, partition, volume, concrete file system or other implementation
  * specific means of file storage. In Linux, these are found in the /proc/mount
  * filesystem, excluding temporary and kernel mounts.
- * 
+ *
  * @author widdis[at]gmail[dot]com
  */
 public class LinuxFileSystem extends AbstractFileSystem {
 
     private static final Logger LOG = LoggerFactory.getLogger(LinuxFileSystem.class);
 
+    // Linux defines a set of virtual file systems
+    private final List<String> pseudofs = Arrays.asList(new String[]{
+        "sysfs", // SysFS file system
+        "proc", // Proc file system
+        "devtmpfs", // Dev temporary file system
+        "devpts", // Dev pseudo terminal devices file system
+        "securityfs", // Kernel security file system
+        "tmpfs", // Temporary file system
+        "cgroup", // Cgroup file system
+        "pstore", // Pstore file system
+        "hugetlbfs", // Huge pages support file system
+        "configfs", // Config file system
+        "selinuxfs", // SELinux file system
+        "systemd-1", // Systemd file system
+        "binfmt_misc", // Binary format support file system
+        "mqueue", // Message queue file system
+        "debugfs", // Debug file system
+        "nfsd", // NFS file system
+        "sunrpc", // Sun RPC file system
+        "fusectl", // FUSE control file system
+        // NOTE: FUSE's fuseblk is not evalued because used as file system representation of a FUSE block storage
+        //"fuseblk" // FUSE block file system
+    });
+    
+    // System path mounted as tmpfs
+    private final List<String> tmpfsPaths = Arrays.asList(new String[]{
+        "/dev/shm",
+        "/run",
+        "/sys/fs/cgroup",
+    });
+
+    private Boolean listElementStarts(List<String> aList, String charSeq) {
+        for (String match : aList) {
+            if (match.startsWith(match)){

```

Comment:
Is this line (`match.startsWith(match)`) correct? You aren't ever checking `charSeq`.